spinnaker/provider_test.go

```diff
@@ -22,10 +22,10 @@ func testAccPreCheck(t *testing.T) {
 	if os.Getenv("GATE_URL") == "" {
 		t.Fatal("GATE_URL must be set for acceptance tests")
 	}
-	err := testAccProvider.Configure(terraform.NewResourceConfig(nil))
-	if err != nil {
-		t.Fatalf("err: %s", err)
-	}
+	//err := testAccProvider.Configure(terraform.NewResourceConfig(nil))
+	//if err != nil {
+	//	t.Fatalf("err: %s", err)
+	//}
```

Comment:
Is it necessary to comment this?

Reply:
@martinohmann Sorry, no I can change this to `NewResourceConfigRaw`
https://www.terraform.io/docs/extend/plugin-sdk.html#deprecations